### ISSUE TITLE
No logging configuration

### ISSUE THREAD

**Issue by brandond** [fugue/emulambda#20]
The Amazon Lambda execution environment provides a root handler that sends all log messages to CloudWatch logs. This is covered [in the documentation](http://docs.aws.amazon.com/lambda/latest/dg/python-logging.html).

emulambda apparently does not set up a base logging environment - any code that assumes a working logger will fail with the less-than-helpful default message that logging spits out when you don't configure it:

```No handlers could be found for logger "root"```

It would be nice if emulambda would at least call ```logging.basicConfig()``` for you, although providing a format string that approximated what you get out of CloudWatch Logs would be even better.

**Comment by dom-at-fugue** [comment]
Hey @brandond -- Sorry for the slow response, things got a little busy here. I'm working on this issue here: https://github.com/fugue/emulambda/tree/feature/logging-and-context. I think to do it right, I will need to also tackle the issue you submitted about the context object. To do _that_ right, I'm gonna need to examine the context code more closely (it was helpfully contributed by someone else, so it'll take me a minute to figure it out).

Hopefully all those will be done this week.

**Comment by brandond** [comment]
@dom-at-fugue FWIW, I ended up doing this: https://github.com/brandond/lambda-harness/

It requires pulling down a few modules from the Lambda environment, but the end result is much closer to the AWS runtime than anything else I've seen.

**Comment by dom-at-fugue** [comment]
@brandond That's a clever idea, to pull stuff down from their runtime. It doesn't get much closer than that.

**Comment by brandond** [comment]
@dom-at-fugue yeah let me know if you have any suggestions. I've avoided going down the road of actually implementing any of the namespace/cgroup stuff that AWS does, and it passes stuff around using pipes instead of shared memory... but it's close enough for testing purposes.
